java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_1/Cello.java

```diff
@@ -0,0 +1,12 @@
+package com.zoolatech.lecture3.tasks._1;
+
+public class Cello extends MusicalInstrument {
```

Comment:
Try to define new classes for objects only if they differ. As for now, there is no difference between classes `Cello`, `Piano` and `Violin` -  you can simply use a single class `MusicalInstrument`.

```
Each instrument has a name and a method to print a unique sound of the instrument.
```

The initial idea was that sound and name are not passed to the constructor - each class defines them.